src/2048-expert/index.js

```diff
@@ -269,23 +274,85 @@ function newGame() {
     clearBoard()
     generateNewTile()
     generateNewTile()
+    addColours()
 }
 
-// Reset Current Score and reset all values to 0
+
+// function clear Board
```

Comment:
Maybe a better comment to describe this function would be "Reset the cells and scores". 
Since the function does more than just clear the board, you could rename this function to something like "resetGame()".

Reply:
I will do this in the next PR